web/ts/global.ts

```diff
@@ -4,6 +4,10 @@ export * from "./notifications";
 export * from "./user-settings";
 export * from "./start-page";
 
+export async function getData(url = "") {
+    return await fetch(url, { method: "GET" });
```

Comment:
It's technically not neeed as far as I know, but maybe we could also specify the content type just to be more explicit and consistent with `putData`.

Reply:
As far as I know is Content-Type not necessary for GET requests since the request body is empty anyways, however, I removed the unnecessary options object ~~`{ method: "GET" }`~~.

Reply:
Yeah, `GET` seems to be default, I also just checked in the console. :+1: